core/src/main/java/org/apache/accumulo/core/client/mapred/AccumuloFileOutputFormat.java

```diff
@@ -186,7 +186,8 @@ public void write(Key key, Value value) throws IOException {
         }
 
         if (out == null) {
-          out = FileOperations.getInstance().openWriter(file.toString(), file.getFileSystem(conf), conf, null, acuConf);
+          out = FileOperations.getInstance().newWriterBuilder().forFile(file.toString(), file.getFileSystem(conf), conf).withTableConfiguration(acuConf)
```

Comment:
Didn't look too closely, but it struck me this could be cleaned up with another constructor that took a Path and  Configuration so it would read
`
out = FileOperations.getInstance().newWriterBuilder().forFile(file, conf).withTableConfiguration(acuConf)
`
Fine as is though